### DISCUSSION TITLE
Enhance video file preview - keyboard navigation and secondary monitor support

### DISCUSSION THREAD

**Discussion by AndiH67** [QL-Win/QuickLook#1302] [Ideas]
QuickLook is indeed a great addition and improvement for the Windows Explorer, and I use it regularly - thanks a lot for making it!

If I may, I would have 2 suggestions ...

I use QuickLook to browse through and organize a large video collection, and it would be great if the video plugin supports navigation hotkeys to skip forward/backward, and to move around - currently one has to use the mouse to control the slider, and cannot use the keyboard alone.  This would be especially useful when browsing media files which have a similar beginning (title/intro).  There was recently another discussion #1291 with a similar idea for audio files :-).  Another option would be an alternative video player plugin (like MPV) that would support hotkeys (seems there was such a plugin in the past, but no longer available - issue #1114).

The other idea would be for systems with dual-monitors: it would be great to have a setting to open the preview window on the secondary monitor and in fullscreen mode (or remember the last position incl. monitor, similar to discussion #1287).

Thanks a lot for consideration!

**Comment by Grooves1210** [comment]
Hotkeys to skip forward/backwards in previewed videos would be great.

I would like arrow keys to remain operational as change to [next/previous] file

Currently, I preview a video, use the mouse to skip forward using the time bar.  But this disables using arrow keys to preview a different file (file explorer loses focus, so can't change to a different file)
